Dockerfile

```
@@ -84,8 +84,8 @@ RUN /install/trivy.sh
 RUN . /.env && \
     F=tools_versions_info && \
     pre-commit --version >> $F && \
-    (if [ "$OPENTOFU_VERSION"       != "false" ]; then echo "./tofu --version | head -n 1" >> $F;      else echo "opentofu SKIPPED" >> $F      ; fi) && \
-    (if [ "$TERRAFORM_VERSION"      != "false" ]; then echo "./terraform --version | head -n 1" >> $F; else echo "terraform SKIPPED" >> $F     ; fi) && \
+    (if [ "$OPENTOFU_VERSION"       != "false" ]; then echo "tofu $(./tofu --version | head -n 1)" >> $F;      else echo "opentofu SKIPPED" >> $F      ; fi) && \
+    (if [ "$TERRAFORM_VERSION"      != "false" ]; then echo "terraform $(./terraform --version | head -n 1)" >> $F; else echo "terraform SKIPPED" >> $F     ; fi) && \
```

The prefixes in output are pointless imho:
```suggestion
    (if [ "$OPENTOFU_VERSION"       != "false" ]; then ./tofu --version | head -n 1 >> $F;      else echo "opentofu SKIPPED" >> $F      ; fi) && \
    (if [ "$TERRAFORM_VERSION"      != "false" ]; then ./terraform --version | head -n 1 >> $F; else echo "terraform SKIPPED" >> $F     ; fi) && \
```
Please adjust indentation of `else ...` part after implementing this suggested change. Thanks.